app/domain/authentication/authn_k8s/kubectl_exec.rb

```diff
@@ -163,30 +187,38 @@ def execute(k8s_object_lookup:,
         stdin: false)
         call(
           k8s_object_lookup: k8s_object_lookup,
-          pod_namespace: pod_namespace,
-          pod_name: pod_name,
-          container: container,
-          cmds: cmds,
-          body: body,
-          stdin: stdin
+          pod_namespace:     pod_namespace,
+          pod_name:          pod_name,
+          container:         container,
+          cmds:              cmds,
+          body:              body,
+          stdin:             stdin
         )
       end
 
-      def copy(k8s_object_lookup:,
-        pod_namespace:,
-        pod_name:,
-        path:,
-        content:,
-        mode:,
-        container: 'authenticator')
+      def copy(k8s_object_lookup:, pod_namespace:, pod_name:, path:, content:, mode:, container: 'authenticator')
+        execute(
+          k8s_object_lookup: k8s_object_lookup,
+          pod_namespace:     pod_namespace,
+          pod_name:          pod_name,
+          container:         container,
+          cmds:              %w(tar xvf - -C /),
+          body:              tar_file_as_string(path, content, mode),
+          stdin:             true
+        )
+      rescue Errors::Authentication::AuthnK8s::CommandTimedOut
+        # The websocket may have not return the response, verify that the file is present in the container
+        @logger.debug(
+          LogMessages::Authentication::AuthnK8s::VerifyCopySSLToPod.new(container, path)
+        )
         execute(
           k8s_object_lookup: k8s_object_lookup,
-          pod_namespace: pod_namespace,
-          pod_name: pod_name,
-          container: container,
-          cmds: ['tar', 'xvf', '-', '-C', '/'],
-          body: tar_file_as_string(path, content, mode),
-          stdin: true
+          pod_namespace:     pod_namespace,
+          pod_name:          pod_name,
+          container:         container,
+          cmds:              %W(ls #{path}),
```

Comment:
Verifying the path is not empty is enough ? We cant have a case we have more than 1 file here?
Need to add test for both behavior (w/o file)